src/database/common.c

```diff
@@ -307,11 +320,15 @@ void db_init(void)
 	}
 
 	logg("Database successfully initialized");
-	database = true;
 }
 
 int db_get_FTL_property(const unsigned int ID)
 {
+	if(!database)
+	{
+		logg("db_get_FTL_property(%u) called but database is not available!", ID);
+		return DB_FAILED;
+	}
```

Comment:
This can be made into a macro to avoid duplicated code.

Reply:
Not really. The functions aren't always the same. Some of them return an integer (`DB_FAILED`), some return a boolean, some are `void`. Also, some of them take an argument, some even two, sometimes unsigned, sometimes signed. I don't think this really justifies any kind of macro.